### PR TITLE
feat(graphql): Added the ability to end a round and draw dice

### LINKED ISSUE #4: [FEAT] Create routes to play the game and do actions on it.
**Is your feature request related to a problem? Please describe.**

Now that we can create games we need an endpoint to be able to play them and do stuff on them.

**Describe the solution you'd like**

Adding a new endpoint to be able to do actions in the game maybe a route per action we can do. 

**Describe alternatives you've considered**

N/A

**Additional context**

N/A

### PR DESCRIPTION
**Pull request by vCertxfiedGoat**
## Describe your changes
Added the ability to draw dice and end a round
## Issue ticket number and link
https://github.com/Carrera-Dev-Consulting/zombie-nomnom-api/issues/4
## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have defined what tests cover my change and or added new test to cover change. 
- [x] I have added comments to the change and updated any relevant docs.
- [x] I have pulled the latest in the main branch and merged them ahead of time with no conflicts.

### CODE REVIEW

File: zombie_nomnom_api/graphql_app/dependencies.py [hunk 1]
```diff
@@ -31,4 +32,6 @@ def get(self, key: Any, default: Any = None) -> Any:
 def bootstrap() -> DIContainer:
     container = DIContainer()
     container[GameMaker] = GameMaker()
+    container[DrawDice] = DrawDice()
+    container[Score] = Score()
```
Comment (gxldCptRick) [inline zombie_nomnom_api/graphql_app/dependencies.py:36]: Disgusting

File: zombie_nomnom_api/graphql_app/resolvers.py [hunk 2]
```diff
@@ -34,6 +35,31 @@ def create_game_resolver(
...
+    maker: GameMaker = dependencies[GameMaker]
+    dice: DrawDice = dependencies[DrawDice]
+    gameInstance: Game = maker[gameId]
+    if gameInstance is None:
+        return {"errors": [f"Game id not found: {gameId}"], "round": None}
+    return {"errors": [], "round": gameInstance.game.process_command(dice)}
+
+
+@Mutation.field("endRound")
+def end_round_resolver(_, __, gameId: str, dependencies: DIContainer = bootstrap()):
```
Comment (gxldCptRick) [inline zombie_nomnom_api/graphql_app/resolvers.py:52]: Eww

File: zombie_nomnom_api/graphql_app/schema.gql [hunk 3]
```diff
@@ -4,13 +4,20 @@ type Query {
 
 type Mutation {
     createGame(players: [String!]!): GameResult!
+    drawDice(gameId: ID!) : RoundResult!
```
Comment (gxldCptRick) [inline zombie_nomnom_api/graphql_app/schema.gql:7]: Simple and clean

**Review by gxldCptRick — CHANGES_REQUESTED** [review]
Where are my goddang integration tests buddy

**Review by Dean-Pesek — APPROVED** [review]
LGTM

**Review by gxldCptRick — APPROVED** [review]
LGTM

**Comment by vCertxfiedGoat** [comment]
> Where are my goddang integration tests buddy

Integration tests were just added 🖕

**Pull request merged**